app/views/auth_key_pair_cloud/_form.html.haml

```diff
@@ -2,8 +2,7 @@
   = _('Basic Information')
   %br
   %br
-
-.form-horizontal
+.form-horizontal{:id => 'start_form_div', :style => 'display:none',	'form-changed' => true}
```

Comment:
@himdel It works after removing `:id => 'start_form_div', :style => 'display:none',` . Did you mean that?